### PR TITLE
Noarch version

### PR DESCRIPTION
**Pull request by HeavyTony2**
Added the missing dependancy xlrd <2.0 and set it as noarch.

Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by HeavyTony2** [comment]
@conda-forge-admin, please rerender

**Comment by HeavyTony2** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,41 +1,44 @@
...
-    - python
+    - python >=3.6 
     - pandas >=0.23.0
     - scipy >=1.4
     - h5py
     - natsort
-    - importlib_metadata  # [py<38]
-    - packaging
+    - importlib_metadata
+    - typing_extensions
```
Comment (flying-sheep) [inline recipe/meta.yaml]: Both of those are only necessary in Python <3.8.

Can you make it so they don’t get installed in Python 3.8+?

**Review by flying-sheep — CHANGES_REQUESTED** [review]
Everything except for conditional dependencies looks good.

**Comment by ivirshup** [comment]
We also no longer rely on xlrd, so it probably shouldn't be added to the dependency list.

I was wondering, does `noarch` imply we can't have python version restrictions?

**Comment by HeavyTony2** [comment]
> We also no longer rely on xlrd, so it probably shouldn't be added to the dependency list.
> 
> I was wondering, does `noarch` imply we can't have python version restrictions?

Sorry for the late answer. The answer is no. In fact, with a noarch version you have to set a python version restriction. The typical one is `python >=3.6` but it could be `python <=4` too for example.

**Comment by HeavyTony2** [comment]
@conda-forge-admin, please rerender

**Pull request closed without merging**